src/client/components/note.vue

```diff
@@ -21,6 +21,12 @@
 			</router-link>
 		</i18n>
 		<div class="info">
+			<span v-if="isMyRenote">
+				<fa :icon="faTimes"/>
+				<span>&nbsp;</span>
+				<a href="#" @click="(e) => { e.preventDefault(); delRenote() }">{{ $t('unrenote') }}</a>
```

Comment:
You can also write here like this, maybe.

```suggestion
				<a href="#" @click.prevent="delRenote()">{{ $t('unrenote') }}</a>
```